codecarbon/emissions_tracker.py

```diff
@@ -189,6 +218,27 @@ def _prepare_emissions_data(self) -> EmissionsData:
             cloud_provider=cloud_provider,
             cloud_region=cloud_region,
         )
+        if delta:
+            if self._previous_emissions is None:
+                self._previous_emissions = total_emissions
+                return total_emissions
+            else:
+                delta_emissions = dataclasses.replace(total_emissions)
+                delta_emissions.duration = (
```

Comment:
maybe we should overload the `-` and `+` operations so that we centralize such manipulations?

Reply:
I've created a method in the class to compute the delta.